### DISCUSSION TITLE
2021-10-26 Website Biweekly Meeting Agenda

### DISCUSSION THREAD

**Discussion by GhadaKandil** [worldiaday/website#43] [Meetings]
Agenda for website meeting tomorrow, Tuesday, 26 October 2021: 
- Current website
  - Home 
  - About 
- Future website 
  - Persona
- Stakeholders' interviews
  - Andrew’s research findings

Join at 7am PST / 4pm CET: https://meet.jit.si/WorldIADayWebsite

**Comment by lauggh** [comment]
Video uploaded: 
https://vimeo.com/639402382/a4afa16f84


See #44 for related action items
